rest/src/server/SubscriptionManager.js

```diff
@@ -93,6 +105,26 @@ class SubscriptionManager {
 			this.delete(channel, client);
 		});
 	}
+
+	/**
+	 * Check if the current subscriber is a cosigner of a multisig account.
+	 * Then subscribe the multisig account for cosigning notification
+	 * @param {string} encodedAddress encloded address of the current subscriber
+	 * @param {object} channel current subscribing channel
+	 * @param {object} client current subscribeing client
+	 */
+	subscribeMultisigAccount(encodedAddress, channel, client) {
```

Comment:
What happens when a cosigner is removed from multisig account? They will still receive notifications from the multisig account until the websocket is close?

Reply:
I don't think it is possible / easy to unsubscribe if cosigner removed from multisig. 
tbh, I don't quite like the idea to hack the rest to subscribe to multisig in ws.  It should be the client to subscribe the multisig account for notification.